src/arkode/arkode_bbdpre.c

```diff
@@ -100,121 +99,113 @@ int ARKBBDPrecInit(void *arkode_mem, sunindextype Nlocal,
   pdata->mlkeep = mlk;
 
   /* Allocate memory for saved Jacobian */
-  pdata->savedJ = SUNBandMatrixStorage(Nlocal, muk, mlk, muk, ark_mem->sunctx);
+  pdata->savedJ = SUNCheckCallLastErrNoRet(SUNBandMatrixStorage(Nlocal, muk, mlk, muk, ARK_SUNCTX));
```

Comment:
I'd like for these `ARK_SUNCTX` to be replaced by `ark_mem->sunctx`, along with the others that were introduced in this PR.

Reply:
OK. I will change these with one big find/replace.